addOns/scanpolicies/src/main/javahelp/help/contents/policy-dev-full.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Developer Full Policy
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Developer Full Policy</H1>
+
+A developer focused policy, including a superset of the dev standard with a greater variety of 
```

Comment:
Worth linking to the dev standard, though less needed if the back link is added.